src/components/promocode-input/index.js

```diff
@@ -17,9 +17,14 @@ import appliedCode from '../../assets/icon-check-circle.svg';
 import styles from "./index.module.scss";
 import { isEmptyString } from '../../utils/utils';
 
-const PromoCodeInput = ({ applyPromoCode, promoCode, removePromoCode, showMultipleTicketTexts }) => {
+const PromoCodeInput = ({ applyPromoCode, promoCode, removePromoCode, showMultipleTicketTexts, onPromoCodeChange }) => {
 
     const [statePromoCode, setStatePromoCode] = useState(promoCode);
+    
+    const handlePromoCodeChange = (value) => {
```

Comment:
this seems that is not needed
u onlyu need to know if the promo code is applied or not